webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -1,10 +1,53 @@
+import browserUpdate from 'browser-update';
 import { isEmpty } from 'lodash';
 import {
   changeOrganization,
   changeLocation,
 } from '../../../foreman_navigation';
 import { translate as __ } from '../../common/I18n';
-import { removeLastSlashFromPath } from '../../common/helpers';
+import {
+  removeLastSlashFromPath,
+  runningInPhantomJS,
+} from '../../common/helpers';
+
+export const checkBrowserSupport = () => {
```

Comment:
nit: layout helper is becoming a large file, how about extract this logig to a separate file ?

Reply:
I have moved that into `browserSupportHelper` in the component directory. As I didn't find any pattern to place components helpers. Or is there one?